docs/Reference/Pantheon-CLI-Syntax.md

```diff
@@ -223,6 +223,25 @@ max-trailing-peers=2
 Specifies the maximum P2P peer connections for peers that are trailing behind the local chain head. 
 The default is unlimited but the number of trailing peers cannot exceed the value specified by [`--max-peers`](#max-peers).
 
+### metrics-enabled
+
+```bash tab="Syntax"
+--metrics-enabled
+```
+
+Set to `true` to enable the [Prometheus](https://prometheus.io/) monitoring service to access Pantheon client metrics.
+The default is `false`.
+
+### metrics-listen
+
+```bash tab="Syntax"
+--metrics-listen
+```
+
+Specifies the host and port on which the [Prometheus](https://prometheus.io/) monitoring service accesses Pantheon
+client metrics. The default is `127.0.0.1:9545`. The metrics server respects the [`--host-whitelist`](#host-whitelist)
+flag for DNS rebind attack protection on RPC.
```

Comment:
Is this point about the metrics server from the implementation point of view?  Does the user need to specify https://prometheus.io/ using the `--host-whitelist` option?  If so, I think that's the information we need to specify here.

Reply:
No, that's not what --host-whitelist is for.  It's not the best name for the flag given it's unclear what the host it is referring to.  It's the name of the host the HTTP service is serving on, so it would be 'localhost' in most cases.  It is not a whitelist of what outside clients can connect. See https://en.wikipedia.org/wiki/DNS_rebinding for what it is designed to mitigate.